src/test/user.spec.js

```diff
@@ -239,7 +287,6 @@ describe('GET api/v1/users', () => {
         expect(res.status).to.equal(200);
         expect(res.body.data).to.be.an('array');
         expect(res.body.data).to.have.lengthOf(4);
```

Comment:
Assert that the items gotten are actually user objects e.g check that they have the required user fields.